docs/api/profiles.md

```diff
@@ -0,0 +1,123 @@
+# Profiles
+
+Profiles contain information about a User. A User does not need a Profile.
```

Comment:
Let's tell why it's useful.

A User does not require a Profile.
For purposes of training the speech models, providing accurate information helps to improve the quality.

Reply:
The text was still from the previous API, I've taken your recommendation and build a bit more upon it.

Reply:
:+1: